slides/body/lect-w01-about.tex

```diff
@@ -255,7 +255,33 @@
 \item [\bfseries Kursansvarig:] Prof. Björn Regnell, bjorn.regnell@cs.lth.se, E:2413
 \item [\bfseries Handledare:]\Emph{teknologer:}\\
 %2021
-\TODO
+\TODO double check this
```

Comment:
Normally I let people themselves provide a PR as a check that they want to be public with their name. So better only put your own name here and I'll remind all that they can get their name here if they want and come with a PR.